src/content/docs/alerts-applied-intelligence/overview.mdx

```diff
@@ -27,15 +27,15 @@ redirects:
 
 import alertsAlertsandAiOverviewPage from 'images/alerts_screenshot-full_alerts-and-ai-overview-page.png'
 
-As the digital world continues to grow more tangled and complex, it's easy for DevOps teams to feel overwhelmed. There are so many new data points, dashboards, entities, and alerts that it can be hard for on-call teams to know what's happening where, and what to do about it. New Relics alerts and applied intelligence were designed to end response fatigue so your team can not only detect and diagnose incidents, but prioritize them for immediate action.
+As the digital world continues to grow more tangled and complex, it's easy for DevOps teams to feel overwhelmed. There are so many new data points, dashboards, entities, and alerts that it can be hard for on-call teams to know what's happening where, and what to do about it. New Relic's alerts and applied intelligence were designed to end response fatigue so your team can not only detect and diagnose incidents, but prioritize them for immediate action.
 
-Alerts allow you to define the conditions in your system you want to be notified about, letting you focus on the things you care about most. Applied intelligence is a machine-learning engine that alerts your team to potential problems by first identifying anomalous behavior, correlating any relevant incidents, alerting your team of active issues and providing a root cause analysis.
+Alerts allow you to define the conditions in your system you want to be notified about, letting you focus on the things you care about most. Applied intelligence is a machine-learning engine that notifies your team about potential problems by first identifying anomalous behavior, correlating any relevant incidents, alerting your team of active issues and providing a root cause analysis.
```

Comment:
I think you can cut out the "first" in "first identifying" since we're not using any other procedural markers. Also should there be a comma between "active issues" and "and providing"? Not sure what our stance on Oxford comma is.